topics/create-first-app.md

```diff
@@ -9,7 +9,13 @@ Here you will learn how to create and run your first KMM application.
     >These steps cannot be performed on other operating systems, such as Microsoft Windows. This is due to an Apple requirement.
     >
     {type="note"}
-    
+
+Now you can either continue reading or watch the video "Kotlin Multiplatform Multiverse, Episode 2: Your First Kotlin 
+Multiplatform Mobile App Tutorial" by Ekaterina Petrova, Kotlin Developer Advocate. The video is pretty much the same 
```

Comment:
The video contains a little bit more information: some diagrams (like the first from [here](https://kotlinlang.org/docs/mobile/connect-to-platform-specific-apis.html) and the first from [here](https://kotlinlang.org/docs/mobile/discover-kmm-project.html)) explaining what's happening, some explanation of the build script, and one more simple example which is not covered in the written tutorial. Also, Kate opens Xcode several times to explain something. 
In many aspects, this video covers the same things regarding the project creation via the wizard, the examples with operating system version, but at the same time, it gives more information and understanding (I hope).

It seems to me that we shouldn't pay much attention to this difference because different people like different means of content. 

And WDYT?

Reply:
It seems that we are talking about the same thing :)
Not pay much attention to the difference + additionally do not put the full video name in the docs :)

Reply:
I felt I need to tell you about the difference just in case. I'll remove the full video name.

Reply:
>I felt I need to tell you about the difference just in case.

Because of this your sentence) Wanted to tell you and remembered now))

>>If the instructions in the video are very similar